### ISSUE TITLE
Code missing ?

### ISSUE THREAD

**Issue by lam560** [vvpossible/homebridge_yeelight#32]

**Comment by TheTherminator** [comment]
you have to set the bulbs with the yeelight app in developer mode / lan mode

**Comment by lam560** [comment]
yes it fix the problem but Yeelight forgot to set the developer mode option in the first release of their redesigned app ...
